docs/howto/mirroring.md

```diff
@@ -0,0 +1,270 @@
+* `MIRROR_NAME` - Name used for the read-only mirror to be created on the destination region
+* `MIRROR_STORAGE_NAMESPACE` - Location acting as the replication target for the storage namespace of our source repository
+
+### Mirroring and Garbage Collection
+
+Garbage collection won't run on mirrored repositories. 
+Deletions from garbage collection should be replicated from the source:
+1. Enable [DELETED marker replication](https://docs.aws.amazon.com/AmazonS3/latest/userguide/delete-marker-replication.html) on the source bucket.
+1. Create a [lifecycle policy](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lifecycle-mgmt.html) on the destination bucket to delete the objects with the DELETED marker.
+
+## RBAC
+
+These are the required RBAC permissions for working with the new cross-region replication feature:
+
+Creating a Mirror:
+
+| Action                     | ARN                                           |
+|----------------------------|-----------------------------------------------|
+| `fs:CreateRepository`      | `arn:lakefs:fs:::repository/{mirrorId}`       |
+| `fs:MirrorRepository`      | `arn:lakefs:fs:::repository/{sourceRepositoryId}` |
+| `fs:AttachStorageNamespace` | `arn:lakefs:fs:::namespace/{storageNamespace}`|
+
+Editing Mirrored Branches:
+
+| Action                  | ARN                                           |
+|-------------------------|-----------------------------------------------|
+| `fs:MirrorRepository`   | `arn:lakefs:fs:::repository/{sourceRepositoryId}` |
+
+Deleting a Mirror:
+
+| Action               | ARN                                     |
+|----------------------|-----------------------------------------|
+| `fs:DeleteRepository`| `arn:lakefs:fs:::repository/{mirrorId}` |
+
+Listing/Getting Mirrors for a Repository:
+
+| Action                | ARN |
+|-----------------------|-----|
+| `fs:ListRepositories` | `*` |
+
+
+## Other replication operations
+
+### Listing all mirrors for a repository
+
+```bash
+curl --location 'https://<ORGANIZATION_ID>.<SOURCE_REGION>.lakefscloud.io/service/replication/v1/repositories/<SOURCE_REPO>/mirrors' \
+-u <ACCESS_KEY_ID>:<SECRET_ACCESS_KEY> -s
+```
+
+### Getting a specific mirror
+
+```bash
+url --location 'https://<ORGANIZATION_ID>.<SOURCE_REGION>.lakefscloud.io/service/replication/v1/repositories/<SOURCE_REPO>/mirrors/<MIRROR_ID>' \
+-u <ACCESS_KEY_ID>:<SECRET_ACCESS_KEY> -s
+```
+
+### Deleting a specific mirror
+
+```bash
+curl --location --request DELETE 'https://<ORGANIZATION_ID>.<SOURCE_REGION>.lakefscloud.io/service/replication/v1/repositories/<SOURCE_REPO>/mirrors/<MIRROR_ID>' \
+-u <ACCESS_KEY_ID>:<SECRET_ACCESS_KEY>
+```
+
+## Limitations
+
+1. Mirroring is currently only supported on [AWS S3](https://aws.amazon.com/s3/) and [lakeFS Cloud for AWS](https://lakefs.cloud)
+1. Read-only mirrors cannot be written to. Mirroring is one-way, from source to destination(s)
```

Comment:
A single mirror is indeed one-way. However I don't think it's clear enough that you can create multiple mirrors from any region to any region